src/features/offer/pages/Offer.tsx

```diff
@@ -42,6 +44,7 @@ const OfferComponent: FunctionComponent = () => {
   } = useRoute<UseRouteType<'Offer'>>()
   const { data: offerResponse } = useOffer({ offerId: dehumanizeId(id) })
   const headerScroll = useRef(new Animated.Value(0)).current
+  const hasSeenAllPage = useRef<boolean>(false)
```

Comment:
```suggestion
  const hasSeenWholePage = useRef<boolean>(false)
```

et pareil pour ConsultWholeOffer, non ?